### PR TITLE
[# 161103616] Benefits dashboard not working for all customers

### PR DESCRIPTION
**Pull request by cobbr2**
Benefits dashboard not working for all customers

[Tracker #161103616](https://www.pivotaltracker.com/story/show/161103616)

The parameter order is important. See https://docs.looker.com/reference/embedding/sso-embed for details.
NOTE: After this is merged, we need to re-publish the gem to Rubygems.

### CODE REVIEW

**Review by soonahn — APPROVED** [review]
Looks good

**Comment by cobbr2** [comment]
Hey, @masonleung and @ajuvekar : Can I get a review from one of you? I'm happy @soonahn likes it, but he doesn't have write permissions to the repo.

**Review by ajuvekar — APPROVED** [review]
LGTM. I just have one question, should the `spec.email` have correct GR email, if so mine should be at grandrounds.com

**Review by nfoen — APPROVED** [review]

**Comment by cobbr2** [comment]
@ajuvekar : `...@grnds.com` *is* a real GrandRounds email address; I use them for developer communications since they're shorter and easier to pronounce that `...@grandrounds.com`.

**Comment by cobbr2** [comment]
After merging, I pushed to RubyGems using the account in  ConsultingMD/engineering/credentials/rubygems.txt.gpg . (rpereira, actually). Transcript:
```
rcobb@rcobb-x250: (master) ~/looker-hmac$ rake release
grnds-looker-hmac 0.1.1 built to pkg/grnds-looker-hmac-0.1.1.gem.
Tagged v0.1.1.
Pushed git commits and tags.
rake aborted!
Your rubygems.org credentials aren't set. Run `gem push` to set them.
/home/rcobb/.rvm/gems/ruby-2.1.5/gems/bundler-1.16.2/lib/bundler/gem_helper.rb:100:in `rubygem_push'
/home/rcobb/.rvm/gems/ruby-2.1.5/gems/bundler-1.16.2/lib/bundler/gem_helper.rb:69:in `block in install'
/home/rcobb/.rvm/gems/ruby-2.1.5/bin/ruby_executable_hooks:15:in `eval'
/home/rcobb/.rvm/gems/ruby-2.1.5/bin/ruby_executable_hooks:15:in `<main>'
Tasks: TOP => release => release:rubygem_push
(See full trace by running task with --trace)
rcobb@rcobb-x250: (master) ~/looker-hmac$ gem push
Enter your RubyGems.org credentials.
Don't have an account yet? Create one at https://rubygems.org/sign_up
   Email:   rick@grnds.com
Password:   ERROR:  Interrupted
rcobb@rcobb-x250: (master) ~/looker-hmac$ gem push
Enter your RubyGems.org credentials.
Don't have an account yet? Create one at https://rubygems.org/sign_up
   Email:   rob.pereira@grandrounds.com
Password:   

Signed in.
ERROR:  While executing gem ... (Gem::CommandLineError)
    Please specify a gem name on the command line (e.g. gem build GEMNAME)
rcobb@rcobb-x250: (master) ~/looker-hmac$ rake release
grnds-looker-hmac 0.1.1 built to pkg/grnds-looker-hmac-0.1.1.gem.
Tag v0.1.1 has already been created.
Pushed grnds-looker-hmac 0.1.1 to rubygems.org
```

**Pull request merged**
